.ci/test_windows.ps1

```diff
@@ -44,6 +44,12 @@ if ($env:TASK -eq "swig") {
 conda init powershell
 conda activate
 conda config --set always_yes yes --set changeps1 no
+
+# ref:
+# * https://stackoverflow.com/a/62897729/3986677
+# * https://github.com/microsoft/LightGBM/issues/5899
+conda install brotlipy
```

Comment:
Seems like maybe the issue described in #5899 was somehow related to a conflict between the `brotli` package installed from PyPI / `conda` and the one that's vendored with `pip`?

https://stackoverflow.com/a/62897729/3986677

I'm not really sure, but this seems to fix it so 🤷🏻

Reply:
oh nice! I still don't understand why this what changed to make this necessary but 🤷🏻